tests/models/test_models.py

```diff
@@ -87,6 +88,23 @@
 }
 
 
+def test_jax_peakvi():
+    n_latent = 5
+
+    adata = synthetic_iid()
```

Comment:
use fixture

Reply:
I'm gonna refactor these tests in a separate PR, I think I can add fixtures for the scvi tests as well